src/project/project.service.ts

```diff
@@ -1121,4 +1149,90 @@ export class ProjectService {
       );
     return userEnrollment.role === ProjectRole.Leader;
   }
+
+  private async getProject(projectId: number): Promise<Project> {
+    const project = await this.projectRepository.findOne({
+      where: { id: projectId },
+      select: ['id', 'userCount', 'requestEnrollmentCount'],
+    });
+    if (!project) throw projectNotFoundError;
+
+    return project;
+  }
+
+  private async getUser(userId: number): Promise<User> {
+    const user = await this.userRepository.findOne({
```

Comment:
I'm feeling it's not a good practice to create methods under Project service that are handling other entities. Responsibility for getting one user shouldn't be under UserService?  Or in case we need to use userRepository I feel like we could go ahead and just use it under ProjectService methods themselves

Reply:
I changed the functio name to `getUserWithInvitations` to make it clear that they are functions associated with _enrollments_ that will only be used in this file and not in `userService`. However, if that is not enough, I can move them to `UserService` and make them _public_ so that they can be used from `ProjectService`, even if they are not used there.